consul/check.py

```diff
@@ -246,18 +246,27 @@ def check(self, instance):
             # Make service checks from health checks for all services in catalog
             health_state = self.consul_request(instance, '/v1/health/state/any')
 
+            sc = {}
+            # compute the highest status level (OK < WARNING < CRITICAL) a a check among all the nodes is running on.
             for check in health_state:
+                sc_id = '{0}-{1}-{2}'.format(check['CheckID'], check.get('ServiceID', ''), check.get('ServiceName', ''))
```

Comment:
wouldn't be better to have `'-'.join([check['CheckID'], check.get('ServiceID', ''), check.get('ServiceName', '')])`? Otherwise you might have ids like `123--`